### ISSUE TITLE
Refactor to use become instead of sudo

### ISSUE THREAD

**Issue by acozine** [curationexperts/ansible-hydra#41]
Sudo is deprecated already. Refactor to use become throughout the roles.

Also, consider which roles if any should run with elevated privileges for all tasks. 
- Advantage of making the entire role run with elevated privileges - code is DRYer. 
- Disadvantage of this approach - may be confusing when looking at the task list, may lead to new tasks running with elevated privileges when they should not, makes debugging by recreating each task at the command line trickier (the tasks themselves don't show that they are being run / need to be run as root). 

Thoughts @hackmastera?

**Comment by mark-dce** [comment]
Let's split this into two tickets:
1) this ticket to replace sudo with become in response to the deprecation (see: http://docs.ansible.com/ansible/become.html#become)
2) a new ticket #44

**Comment by mark-dce** [comment]
I missed something in #46. I'm getting a series of errors like

```
./ansible.log:2015-12-04 12:43:52,950 p=72953 u=mark |  ERROR: incompatible parameters ("become", "become_user", "become_pass") and sudo params "sudo", "sudo_user", "sudo_pass" in task: install passenger-install-apache2-module
./ansible.log:2015-12-04 13:49:26,317 p=73181 u=mark |  ERROR: incompatible parameters ("become", "become_user", "become_pass") and sudo params "sudo", "sudo_user", "sudo_pass" in task: symlink passenger config
./ansible.log:2015-12-04 13:49:42,294 p=73231 u=mark |  ERROR: incompatible parameters ("become", "become_user", "become_pass") and sudo params "sudo", "sudo_user", "sudo_pass" in task: restart ssh service
```

I'll investigate and submit an updated PR

**Comment by mark-dce** [comment]
Ok, figured it out. The playbook in my repo (https://github.com/mark-dce/sufia-prod-ubuntu-vagrant/blob/9abf1cda8c542eff25517584cedf1b2338700753/vagrant-sufia-prod.yml) still had role-level sudo, and those roles have task level sudo/become directives as well.  I've updated my calling playbook to use become and the errors are no longer occurring.

**Comment by acozine** [comment]
Closed by 48d7a57a2b02636699aaf81689e1bdae12cfbcde.
